### ISSUE TITLE
Time between sibling births

### ISSUE THREAD

**Issue by kirkstall** [ShammyLevva/FTAnalyzer#318]
I have an adopted child, which is marked as such, who has a birthday within 40 days of a biological child of the parents. This has been flagged as an error which it obvioulsy isn't.
Also the time period for child likely born too soon after sibling seems to be too long to me. Bilologically it can be as little as 280 days. I suspect the test is currently set at 365 days. I would suggest 300 or 320 days.

**Comment by ShammyLevva** [comment]
What does the GEDCOM say for the adopted child? How has the program you are using recorded that. Can you create a GEDCOM snippet so I can see what's going on please. It should ignore adoption for that error. So it suggests the GEDCOM is not recording the type of child as adopted or is recording it in a manner I didn't cater for.

The too soon was 365 I've reduced it to 300 as per suggestion.

**Comment by kirkstall** [comment]
Here it is. Let me know if you want the GEDCOM of the family. THe GRDCOM was created using Family Historian v7

0 @I9684@ INDI
1 NAME Clarice /Peat/
1 SEX F
1 BIRT
2 DATE 27 JUN 1891
2 PLAC Northwich,CHS,ENG
1 CENS
2 DATE 31 MAR 1901
2 PLAC Saddleworth,YKS,ENG
2 ADDR Dobcross
2 SOUR @S1375@
2 AGE 8y
1 CENS
2 DATE 2 APR 1911
2 PLAC Saddleworth,YKS,ENG
2 ADDR Dobcross
2 SOUR @S1376@
2 AGE 19y
1 OCCU Shawl weaver
2 DATE 2 APR 1911
2 PLAC Saddleworth,YKS,ENG
2 NOTE Worker
2 SOUR @S1376@
1 CENS
2 DATE 19 JUN 1921
2 PLAC Dobcross,YKS,ENG
2 ADDR Meadow Bank, Woods Lane
2 SOUR @S2940@
2 AGE 29y 11m
1 OCCU Home Duties
2 DATE 19 JUN 1921
2 PLAC Dobcross,YKS,ENG
2 SOUR @S2940@
1 DEAT
2 DATE BET JUL 1967 AND SEP 1967
2 PLAC Nelson,LAN,ENG
2 SOUR @S305@
3 PAGE Nelson 1967 Q3 10e 442
2 AGE 76y
1 FAMC @F1720@
2 PEDI Adopted
1 FAMS @F2217@
1 _FLGS
2 __UK_CENSUS_190 UK Census 1901
2 __UK_CENSUS_191 UK Census 1911
1 CHAN
2 DATE 4 NOV 2023
3 TIME 08:17:59

**Comment by ShammyLevva** [comment]
Yes whilst there is a PEDI line after the FAMC the family record will hold the link between parent and child. I don't seem to use that only the family record. So if I can see what is in the family record i can see why it's not picking up this PEDI record. So yes if you could supply the family record, specifically in this case Family 1720.

**Comment by kirkstall** [comment]
There does not seem to be anything there regarding adoption.

0 @F1720@ FAM
1 MARR
2 DATE 3 AUG 1867
2 PLAC Saddleworth,YKS,ENG
2 SOUR @S309@
3 PAGE Saddleworth 1867 Q3 9a 308
1 HUSB @I7630@
1 WIFE @I9656@
1 CHIL @I9657@
1 CHIL @I9661@
1 CHIL @I9658@
1 CHIL @I9659@
1 CHIL @I9660@
1 CHIL @I9662@
1 CHIL @I9663@
1 CHIL @I9665@
1 CHIL @I9664@
1 CHIL @I9666@
1 CHIL @I9684@
1 CHAN
2 DATE 21 NOV 2016
3 TIME 09:16:17
0 @F1722@ FAM
1 MARR

**Comment by kirkstall** [comment]
I have been looking at other adopted individuals in my tree. Some have 1 ADOP. Do you test for this? If you do then I think I can add it.

**Comment by ShammyLevva** [comment]
Yes there is a test for a specific ADOP flag. However I'm not then sure that is used in the Natural Parent filters. There seems to be a bit of a mismatch in the way the GEDCOM spec is setup. There's a note in the specification that there was a change in v5.5 to move the tag from the family to the individual but suggesting the individual should have a indicator of who undertook the adoption ie: father, mother or both. As it is entirely plausible (likely for early adoptions?) that the adoption was one sided ie: the child was a natural child of one of the parents and the other adopted them. So flagging the child as adopted by both could be misleading.

I think this needs a bit more thought as to how best to treat it. It's especially difficult when the specification is a bit vague.

**Comment by ShammyLevva** [comment]
From the GEDCOM specification document.

> Adoption Events:
In GEDCOM 5.x, the ADOPtion event was moved from the FAM.CHIL structure to the
INDI.FAMC.ADOP structure, it also appears in the INDI.ADOP structure. In GEDCOM 5.4 the
ADOPtion event appears only as an individual event which optionally contains a FAMC pointer to the
adoptive family. Subordinate to this pointer is another ADOPtion tag which indicates whether the
HUSB or WIFE in the pointed at family was the adoptive parent (see
<ADOPTED_BY_WHICH_PARENTS> primitive on page 42). Pedigree navigation is provided only
by <<CHILD_TO_FAMILY_LINK>> structure found on page 31

**Comment by kirkstall** [comment]
I have read up on the Family Historian forum and it seems to confirm your ideas. In my record where ADOP appears I do know at least one of the biological parents but not in the case in point. 
I will force  ADOP in the record and let you know the outcome in the morning.

**Comment by kirkstall** [comment]
Forcing ADOP in the individual record appears not to work,

**Comment by ShammyLevva** [comment]
I'm going to have to have a think about how to approach this. A FAMC tag that uses the flag adopted is an accepted way of flagging an adoption but it's not terribly clear if that means that neither parent is the biological parent as there is no way in that tag that one parent can be flagged as natural. 

So with either ADOP tag or FAMC tag as you originally had I'd need to assume neither parent was biological. The program doesn't currently do that. That would have the effect of treating the parents, siblings and all generations further back as unrelated. I'm not entirely sure that is a wise course of action as many people who are adopted treat their adopted family as just as much their family as their biological one and so cutting the branch as it were may not be the expected treatment.
